src/main/java/duke/utility/Storage.java

```diff
@@ -0,0 +1,204 @@
+
+            //Reading all the lines of input text file into oldContent
+            String line = reader.readLine();
+
+            while (line != null) {
+                // do not copy contents if it is equals to the task to delete
+                if (!line.equals(oldTask)) {
+                    newContent += line + System.lineSeparator();
+                }
+                line = reader.readLine();
+            }
+
+            //Rewriting the input text file with newContent
+            writer = new FileWriter(fileToUpdate);
+
+            writer.write(newContent);
+        } catch (IOException e) {
+            e.printStackTrace();
+        } finally {
+            try {
+                //Closing the resources
+                assert reader != null;
+                reader.close();
+                assert writer != null;
+                writer.close();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    /**
+     * Loads all contents from the text file into the active task list.
+     *
+     * @param list Active task list to load into.
+     */
+    public void handleLoad(TaskList list) {
+        Scanner sc = null;
+        try {
+            File file = new File(this.pathString);
+
+            file.getParentFile().mkdirs();
+
+            // creates new file if it does not exist
+            if (!file.exists()) {
+                file.createNewFile();
+            }
+
+            sc = new Scanner(file);
+
+            while (sc.hasNext()) {
+                // reads text file line by line
+                String[] keyword = sc.nextLine().split("/@/");
+                switch (keyword[0]) {
+                    case "T":
+                        // line is a To-Do Task
+                        boolean tStatus = keyword[1].equals("1");
+                        list.addTask(new ToDo(tStatus, keyword[2]));
+                        break;
+                    case "D":
+                        // line is a Deadline Task
+                        boolean dStatus = keyword[1].equals("1");
+                        list.addTask(new Deadline(
+                                dStatus,
+                                keyword[2],
+                                LocalDateTime.parse(keyword[3])
+                        ));
+                        break;
+                    case "E":
+                        // line is an Event Task
+                        boolean eStatus = keyword[1].equals("1");
+                        list.addTask(new Event(
+                                eStatus,
+                                keyword[2],
+                                LocalDateTime.parse(keyword[3]),
+                                LocalDateTime.parse(keyword[4])
+                        ));
+                        break;
+                    default:
+                        System.out.println("error");
+                        break;
+                }
```

Comment:
Small thing, but I believe the course conventions for switch is to have no indentation for case clauses.